modules/openapi-generator/src/main/resources/ruby-client/gitlab-ci.mustache

```diff
@@ -0,0 +1,26 @@
+.ruby: &ruby
+  variables:
+    LANG: "C.UTF-8"
+  before_script:
+    - ruby -v
+    - bundle config set --local deployment true
+    - bundle install -j $(nproc)
+  parallel:
+    matrix:
+      - RUBY_VERSION: ['2.3', '2.4', '2.5', '2.6', '2.7', '3.0']
+  image: "ruby:$RUBY_VERSION"
+  cache:
+    paths:
+      - vendor/ruby
+    key: 'ruby-$RUBY_VERSION'
+
+gem:
+  extends: .ruby
+  script:
+    - bundle exec rspec
+    - bundle exec rake build
+    - bundle exec rake install
+  artifacts:
+    paths:
+      - pkg/*.gem
```

Comment:
Uses rake tasks from Rubygems instead of calling `gem build` and `gem install` manually.

This has the benefit of getting generated files inside `pkg` folder which can be exposed easily as an artifact after the build.